RFC0000-RFC-Process.md

```diff
@@ -62,59 +62,53 @@ Description and rationale.
 
 ## RFC Workflow
 
-RFCs go through applicable stages:
+RFCs may go through the following stages:
 
-* Draft
+### Draft
+ 
+This is the initial draft of an RFC posted for comments and considered a work-in-progress.
 
-This is the initial draft of a RFC posted for comments and considered a work-in-progress.
+* New proposed drafts should be submitted as a Pull Request from the Author's fork into the `Draft-Accepted` folder.
+* The Author shall ensure that the `Allow edits from maintainers` box is checked so that a Committee Member can assign the next RFC number.
+* If the Pull Request has followed the correct template and process, a Committee Member will assign the label `Review - Open for comments` to the Pull Request to indicate that anyone can comment on the content of the submission.
+Typically, one or two months is allowed for comments, though this may be extended if a submission is particularly conentious or hasn't received enough feedback for the Committee to feel comfortable making a decision.
+* When the Committee closes the comment period, the Author should update the RFC and Pull Request to address the comments.
+* The Committee shall vote to accept or reject the RFC and accept/close the Pull Request accordingly.
 
-New proposed drafts should be submitted as a Pull Request from the Author's fork.
-The Author must ensure that 'Allow edits from maintainers' is checked so that a Committee Member can assign the next RFC number.
+### Draft-Accepted
 
-Committee Members also ensure that the RFC adheres to the template and the RFC process before merging the Pull Request.
+The PowerShell Committee has reviewed the RFC and comments, and has voted to accept the RFC as a Draft.
 
-After the Pull Request is merged, Maintainers will ask the Author to create an issue for this RFC to collect and respond to feedback on the RFC.
-(The Author should create issue so that they get notified of new comments instead of the Committee Member.)
-Typically, one or two months is allowed for comments (see `Comments Due` above).
+* New comments are not being sought.
+* No one has begun implementing the RFC, and there are no current plans to implement the RFC.
+* The community is invited to implement the RFC (unless explicitly stated in the RFC).
```

Comment:
I think that the default assumption should be that the proposer is planning to implement the RFC.  Otherwise, the RFC's can end up as a bunch of unfunded mandates.  If you want to open the implementation up, that would fit in issues which there is a mechanism for tagging things that are up for grabs.